pkg/types/azure/validation/platform.go

```diff
@@ -1,8 +1,11 @@
 package validation
 
 import (
+	"regexp"
+
 	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/azure"
+	"github.com/pkg/errors"
```

Comment:
generally we follow a pattern for imports which is:
```
[stdlib packages]
new line
[external (non installer) packages]
new line
[internal (installer) packages]
```
so this should be: 

```
        "regexp"

	"github.com/pkg/errors"
	"k8s.io/apimachinery/pkg/util/validation/field"

	"github.com/openshift/installer/pkg/types"
	"github.com/openshift/installer/pkg/types/azure"
```
 and yes `"k8s.io/apimachinery/pkg/util/validation/field"` was out of order before your changes.